apps/camera/js/views/controls.js

```diff
@@ -245,11 +250,19 @@ module.exports = View.extend({
     this.unset(key ? key + '-enabled' : 'enabled');
   },
 
+  localize: function() {
+    for (var el in this.elsL10n) {
+      this.els[el].setAttribute('data-l10n-id', this.elsL10n[el]);
```

Comment:
No, my question was referring to the fact that I don't see where `this.elsL10n` gets localized values.